### PR TITLE
DOC: Moved the sort_values doc from generic.py to frame.py

### PR DESCRIPTION
**Pull request by F-Canaboi**
Hello,

this is my first pull request to pandas and on GitHub in general.
While looking into an issue in sort_values in frame.py I saw the
TODO about moving the sort_values doc and did it.

Cheers, Felix

- [ ] closes #xxxx (Replace xxxx with the GitHub issue number)
- [ ] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [ ] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [ ] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

### CODE REVIEW

File: pandas/core/frame.py [hunk 1]
```diff
@@ -6814,7 +6814,6 @@ def sort_values(
     ) -> None:
         ...
 
-    # TODO: Just move the sort_values doc here.
     # error: Signature of "sort_values" incompatible with supertype "NDFrame"
     @Substitution(**_shared_doc_kwargs)
     @Appender(NDFrame.sort_values.__doc__)
```
Comment (mroeschke) [inline pandas/core/frame.py:6819]: I _think_ this can be removed too

**Comment by mroeschke** [comment]
Thanks. Could you build the docs locally and share a screenshot of how the docs look after this change? https://pandas.pydata.org/pandas-docs/stable/development/contributing_documentation.html

File: pandas/core/frame.py [hunk 2]
```diff
@@ -6814,7 +6814,6 @@ def sort_values(
     ) -> None:
         ...
 
-    # TODO: Just move the sort_values doc here.
     # error: Signature of "sort_values" incompatible with supertype "NDFrame"
     @Substitution(**_shared_doc_kwargs)
```
Comment (mroeschke) [inline pandas/core/frame.py:6818]: Also whatever values are being substituted with this call should just be in-lined in the docstring

Comment (F-Canaboi) [reply pandas/core/frame.py:6704]: Hello,
I want to apologize for replying only now. I first encountered problems with creating a development environment for building the documentation locally and then got struck down with Covid.

I'm now back on track and will take care of this.

Cheers

Comment (mroeschke) [reply pandas/core/frame.py:6704]: No worries (and no need to apologize). Glad you are feeling better!

**Comment by MarcoGorelli** [comment]
Thanks for your PR - looks like this is no longer active (no activity in > 1 month), so I'll close it for now

If you still interested in working on this and addressing the review comments, please let us know and we can reopen

**Pull request closed without merging**